packages/api/src/server/search/rover.response.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class Rover {
```

Comment:
Is there any specific reason why you created this class?  as per the fields, it looks more like User and less like Rover

Reply:
Actually It'll transform the data which we search form rover, but the class name can be changed.